codestyle-config/pom.xml

```diff
@@ -22,16 +22,18 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
   <parent>
-    <groupId>org.apache.apex</groupId>
```

Comment:
I can move them into a separate commit, but we can't combine changes to the code style and the parent pom into a single PR.